src/py/flwr/client/rest_client/connection.py

```diff
@@ -60,12 +62,15 @@ def http_request_response(
     Parameters
     ----------
     server_address : str
-        The IPv6 address of the server. If the Flower server runs on the same machine
-        on port 8080, then `server_address` would be `"[::]:8080"`.
+        The IPv6 address of the server with `http://` or `https://`.
+        If the Flower server runs on the same machine
+        on port 8080, then `server_address` would be `"http://[::]:8080"`.
     max_message_length : int
         Ignored, only present to preserve API-compatibility.
-    root_certificates : Optional[bytes] (default: None)
-        Ignored, for now.
+    root_certificates : Optional[Union[bytes, str]] (default: None)
+        Path of the root certificate. If provided, a secure
+        connection using the certificates will be established to a SSL-enabled
+        Flower server. Bytes won't work for the REST API.
```

Comment:
We can get rid of this if we change the type:
```suggestion
        Flower server.
```

Reply:
@danieljanes we could but it would be a breaking change here.